hips/utils/wcs.py

```diff
@@ -33,17 +34,53 @@ def wcs(self) -> None:
         return self._wcs
 
     @classmethod
-    def create(cls, projection: str='AIT', cdelt: float=1.0, crpix: float=1., shape: tuple) -> 'WCSGeometry':
+    def create(cls, skydir: SkyCoord, shape: tuple, coordsys: str='CEL', projection: str='AIT', cdelt: float=1.0, crpix: float=1.) -> 'WCSGeometry':
         """Read from HiPS description file (`WCSGeometry`).
 
         Parameters
         ----------
-        projection : `str`
-            Project of the WCS object
-        cdelt : `str`
-        crpix : `str`
+        skydir : `~astropy.coordinates.SkyCoord`
+            Sky coordinate of the WCS reference point
         shape : `tuple`
-            Image shape
+            Shape of the image
+        coordsys : `str`
+            Coordinate system
+        projection : `str`
+            Projection of the WCS object
+        cdelt : `float`
+            Coordinate increment at reference point
+        crpix : `float`
+            Pixel coordinate of reference point
         """
-        # TODO
-        pass
+
+        naxis = 2
+        if shape is not None:
```

Comment:
At least for now, we only process 2-dim images.
So here you can just remove the part computing naxis from shape, and instead just do this:
```
w = WCS(naxis=2)
```
and then the WCS transforms should work as you know them from other examples / your notebooks